src/Control/Effect/Sum.hs

```diff
@@ -30,3 +37,9 @@ instance {-# OVERLAPPABLE #-} Member sub sup => Member sub (sub' :+: sup) where
   inj = R . inj
   prj (R g) = prj g
   prj _     = Nothing
+
+
+-- | Construct a request for an effect to be interpreted by some handler later on.
+send :: (Member effect sig, Carrier sig m) => effect m a -> m a
+send = eff . inj
+{-# INLINE send #-}
```

Comment:
Glad to have this, and the `HFunctor`/`Effect` instances, back where they belong.